core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2700,11 +2735,13 @@ private void setUfsAcl(LockedInodePath inodePath)
             + "UFS: " + ufsUri + ". This has no effect on the underlying object.");
       } else {
         try {
-          ufs.setAcl(ufsUri, inode.getACL());
+          List<AclEntry> entries = new ArrayList<>();
```

Comment:
can this be sized to the correct size (or at least be initialized with `inode.getACL().getEntries()`?

Reply:
done